index.test.ts

```diff
@@ -0,0 +1,14 @@
+import {TableMap, TableRect} from 'prosemirror-tables';
```

Comment:
Should we use `../src` instead?

Reply:
The types are not coming from the source code,  they are coming from the modules declaration `index.d.ts`